src/main-process/agent-handlers.ts

```diff
@@ -31,6 +31,7 @@ import {
   refreshConnectedStatus,
 } from './connected-status';
 import {
+  CALLBACK_URL,
```

Comment:
why is that? for testing?

Reply:
Yes, for uploading playwright attachments (screenshots etc.) in dev env.